doc/plugins/org.polarsys.capella.tipsandtricks.doc/html/Tips and tricks/6_Diagram exploration/6_Diagram exploration.html

```diff
@@ -233,5 +233,19 @@ <h2 id="Refresh_many_representations_at_once">Refresh many representations at on
 				</div>
 			</div>
 		</p>
+		<h2 id="Title_Blocks">Title Blocks</h2>
+		<p>The title blocks enables the possibility to add information about diagram elements or the diagram itself and can be shown in diagram to provide additional information chosen by users.
+			Additional details about title blocks (tools, customization) can be found below:</p>
+		<ul>
+			<li>
+				<a href="https://wiki.corp.thales/pages/viewpage.action?pageId=785879073#id-2020What&apos;snewinCapella5.0?-TitleBlocks">Capella what's new page</a>
```

Comment:
We can't use an intranet link on opensource ;)
we can use https://github.com/eclipse/capella/wiki/Release-Notes-1.4.1 instead

Reply:
Ok.